source/documentation/tdb/optimizer.md

```diff
@@ -65,18 +65,20 @@ Optimizer control files
 
 | File name   | Effect   |
 | ----------- | -------- |
-| `none.opt`  | No reordering - execute triple patterns in the order in the query |
```

Comment:
Please leave `none.opt` in.

The issue is that there is reordering in the optimizer as well which helps put filter in better places. `none.opt` still applies at the TDB level.